test/integration/enumerating-databases/list_databases.test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../../src';
+
+describe.only('listDatabases', function () {
```

Comment:
```suggestion
describe('listDatabases', function () {
```

Let's give the other tests a chance to run! 🚀  😁